### DISCUSSION TITLE
Freeze" the visualized model in the current position

### DISCUSSION THREAD

**Discussion by NitinJ88** [google/model-viewer#4861] [General]
Is it possible to "freeze" the visualized model in the current position so that the user cannot accidentally move the model from the current position?

**Comment by elalish** [comment]
In AR, no; in 3D yes - `mv.cameraControls = false;`. What's your use case?

**Reply by NitinJ88** [reply]
if we can set the size of a 3D model to its actual size (size from 3D CAD). For example, if I visualize a 3D model of a table that is 1x1meter high and 2x2meter wide, currently there is no way to display it in true scale relative to the surrounding objects.

**Reply by elalish** [reply]
That's not true - we display models are true scale in AR - glTF defines it's units as always meters. Can you show a screenshot / URL where it's not working as expected?
